sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -80,6 +82,28 @@ def show(self, name):
         except BaseException:
             return None
 
+    def _get_statuses(self, batch_ids):
+        try:
+            url = "http://{}/{}".format(self._base_url, 'batch_status')
+
+            if isinstance(batch_ids, bytes):
+                headers = {'Content-Type': 'application/octet-stream'}
```

Comment:
`application/octet-stream` is only for a `POST /batches`. A `POST /batch_status` will _only_ accept `application/json`. So even if somebody submitted `batch_ids` as bytes, you would only get back a `400` error.